### ISSUE TITLE
Comparison with pnpm for benchmarks

### ISSUE THREAD

**Issue by aidenybai** [voltpkg/volt#13]
It would be very helpful if pnpm was included in the benchmarks

**Comment by XtremeDevX** [comment]
@aidenybai Yes, this is something I've been looking into. I'll re-benchmark the code with a large `package.json` file to emulate something a bit more realistic. I'll be sure to add `pnpm` to those benchmarks.

**Comment by re-thc** [comment]
Why not use https://github.com/pnpm/pnpm.github.io/tree/main/benchmarks ? It's constantly updated in https://pnpm.io/benchmarks.

**Comment by shellscape** [comment]
cc @zkochan

**Comment by XtremeDevX** [comment]
> @aidenybai Yes, this is something I've been looking into. I'll re-benchmark the code with a large `package.json` file to emulate something a bit more realistic. I'll be sure to add `pnpm` to those benchmarks.

Just an update on this. I'd like to update the benchmarks as soon as I've finished the `add` and `install` commands to ensure fair benchmarks. Take the current benchmarks with a grain of salt, because, as I said, volt is still in the development stage and there might be improvements or slowdowns as the codebase grows.

**Comment by Milo123459** [comment]
Any updates?
